CONTRIBUTING.md

```diff
@@ -95,6 +96,98 @@ have more control over the process and debug any issues.
 1. Run `rush change` to generate changelog. You will need to follow the interactive prompts.
    You can edit the added files later. If you don't add the right entries, the CI build will fail.
 
+# How to add and roll out new linter rules
+
+This section describes the process for adding a new rule to a ruleset. The rule will first be added to
+the staging pipeline, where it will not affect the ability to merge a PR. While the rule is in the
+staging pipeline, the author can verify that the rule is working correctly and not incorrectly marking
+violations. After verifying, the author can add the rule to the production pipeline so that the specs repo
+pipeline will block specs that violate the new rule from merging into any of the main/production branches.
+
+1. Ensure the new rule is set to run [only in staging](#how-to-set-a-spectral-rule-to-run-only-in-staging)
+1. Merge the new rule to the main branch. Once merged, your new rule will start running in the staging pipeline. You can
+verify the rule is running with the instructions in [Verify the deployed changes](#verify-the-deployed-changes).
+1. Review Staging LintDiff pipeline build logs to see if the rules work correctly.
+    1. Use Kusto to find PRs that ran your new rule:
```

Comment:
> Use Kusto to find PRs that ran your new rule: [](http://example.com/codeflow?start=6&length=46)

Can u also include the details of the kusto cluster and link the doc that talks about how to get access.

Reply:
That info is here:

- https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/625/Kusto-Query-Catalog?anchor=cluster-connection-strings

We can link to it. But I think it would be also fine to duplicate the relevant subset of information in this doc.